### PR TITLE
chore(mongodb): migrate mongodb examples

### LINKED ISSUE #1158: Migrate examples for mongodb instrumentation
We're moving instrumentation examples from the top-level examples folder to the individual instrumentation packages to help keep them up to date. #939 

This issue is specifically for the [@opentelemetry/instrumentation-mongodb](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/examples/mongodb) examples.

Step-by-step instructions for migrating these examples can be found in [examples/README.md](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/examples/README.md#instructions-for-migrating-an-example).

Please assign this to me :)

### PR DESCRIPTION
**Pull request by osherv**
## Which problem is this PR solving?

Fixed #1158 

## Short description of the changes

Migrated MongoDB examples to the MongoDB instrumentation package.

### CODE REVIEW

File: plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    })
+  });
+
+  provider.addSpanProcessor(new SimpleSpanProcessor(new ZipkinExporter()));
+  provider.addSpanProcessor(new SimpleSpanProcessor(new JaegerExporter()));
+
+  // Initialize the OpenTelemetry APIs to use the NodeTracerProvider bindings
+  provider.register();
+
+  registerInstrumentations({
```
Comment (vmarchaud) [inline plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts:27]: you should move the instrumentation above zipkin/jaeger as its best practices to create them before anything else

File: plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    instrumentations: [
+      new HttpInstrumentation(),
+      new MongoDBInstrumentation({
+        enhancedDatabaseReporting: true,
+
```
Comment (vmarchaud) [inline plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts:32]: empty line

File: plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      }),
+    ],
+    tracerProvider: provider,
+  });
+
+  return api.trace.getTracer('mysql-example');
```
Comment (vmarchaud) [inline plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts:38]:
```suggestion
  return api.trace.getTracer('mongodb-example');
```

File: plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts (hunk 1 above)
Comment (osherv) [reply plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts:27]: Fixed thanks

File: plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts (hunk 2 above)
Comment (osherv) [reply plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts:32]: Fixed, thanks

File: plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts (hunk 3 above)
Comment (osherv) [reply plugins/node/opentelemetry-instrumentation-mongodb/examples/src/tracer.ts:38]: O_o fixed, thanks!

**Review by vmarchaud — APPROVED** [review]

**Review by dyladan — APPROVED** [review]

**Comment by dyladan** [comment]
can't merge because out of date

**Comment by vmarchaud** [comment]
@osherv Sorry to bother again but could you rebase once again ? I've merged another one

**Comment by osherv** [comment]
@vmarchaud Sure! I'm sry. I moving to a personal fork so you can merge next time :) Thanks

**Pull request merged**